src/SFA.DAS.AANHub.Application.UnitTests/SFA.DAS.AANHub.Application.UnitTests.csproj

```diff
@@ -38,6 +38,7 @@
 
   <ItemGroup>
     <Folder Include="Apprentices\Commands\" />
+    <Folder Include="Employers\Queries\" />
```

Comment:
Not sure why these Folder attributes are getting into project file, if there are files in the folder then this should be removed.